packages/twenty-server/src/modules/messaging/message-import-manager/drivers/microsoft/services/microsoft-get-message-list.service.ts

```diff
@@ -195,7 +200,12 @@ export class MicrosoftGetMessageListService {
       .headers({
         Prefer: `odata.maxpagesize=${MESSAGING_MICROSOFT_USERS_MESSAGES_LIST_MAX_RESULT}, IdType="ImmutableId"`,
       })
-      .get();
+      .get()
+      .catch((error) => {
+        this.microsoftHandleErrorService.handleMicrosoftMessageFetchError(
```

Comment:
LGTM, but let's introduce handleMicrosoftMessageListFetchError and handleMicrosoftMessagesImportError functions within microsoftHandleErrorService